### ISSUE TITLE
Remove 'Skip to content' link in foundation_access_iframe theme

### ISSUE THREAD

**Issue by heyMP** [elmsln/elmsln#281]
When tabing through content, if there is a token that has been rendered in an entity_iframe display, the 'skip to content' link will reveal itself in the iframed content.  This should be removed from the iframe theme.

![screen shot 2015-10-08 at 2 32 45 pm](https://cloud.githubusercontent.com/assets/3428964/10376050/dfd4bc16-6dc9-11e5-8e10-6a305c566525.png)

**Comment by heyMP** [comment]
@mmilutinovic1313 This is one that you can probably add to your list of possible issues to tackle.

**Comment by btopro** [comment]
Look for foundation_access_iframe as this is based on fa but strips a ton off

Wp8
________________________________
